src/constants.js

```diff
@@ -173,3 +173,13 @@ export const HASH_KEY_SIZE = 256;
  * Return code of the send_tokens response when there is a stratum timeout
  */
 export const STRATUM_TIMEOUT_RETURN_CODE = 'stratum_timeout';
+
+/**
+ * Minimum job status poll to update job data when mining a tx
+ */
+export const MIN_POLLING_INTERVAL = 0.5;
+
+/**
+ * Base URL for the tx mining api
+ */
+export const TX_MINING_URL = 'http://localhost:8001/';
```

Comment:
Need to update this before merging. Do we have a live test API already?

Reply:
We do. The live test API is available at http://txmining.mainnet.hathor.network/.

Reply:
Updated